modules/react/button/lib/parts/ButtonLabelIcon.tsx

```diff
@@ -5,12 +5,15 @@ import isPropValid from '@emotion/is-prop-valid';
 import {SystemIcon} from '@workday/canvas-kit-react/icon';
 import {styled} from '@workday/canvas-kit-react/common';
 
+type ButtonSizes = 'extraSmall' | 'small' | 'medium' | 'large';
```

Comment:
Also our types/enums are generally singular

Reply:
We've been moving toward plural since V5 – we've even added codemods for that update with tokens. I agree that we should standardize, but even in `button/lib/types` we have `ButtonStateColors` and `ButtonColors`. 🤷🏻 

https://github.com/Workday/canvas-kit/blob/master/modules/react/button/lib/types.ts

I'll move it to the types file and keep it plural for now. Happy to talk about that more later. 👍🏻